aiogram/types/inline_keyboard.py

```diff
@@ -82,6 +83,17 @@ def insert(self, button):
             self.add(button)
         return self
 
+    def __iadd__(self, other):
+        return self.add(other)
+
+    def __add__(self, other):
+        return self.__class__(
+            inline_keyboard=self.inline_keyboard + other.keyboard,
```

Comment:
`other` has no `keyboard` attribute, use `inline_keyboard` instead

Reply:
thank you, fixed